crates/proto/src/generated/responses.rs

```diff
@@ -233,14 +229,11 @@ pub struct AccountProofsResponse {
     /// Account ID.
     #[prost(message, optional, tag = "1")]
     pub account_id: ::core::option::Option<super::account::AccountId>,
-    /// Account commitment.
+    /// Proof of inclusion for the account's current state commitment in the account tree.
     #[prost(message, optional, tag = "2")]
-    pub account_commitment: ::core::option::Option<super::digest::Digest>,
-    /// Authentication path from the `account_root` of the block header to the account.
-    #[prost(message, optional, tag = "3")]
-    pub account_proof: ::core::option::Option<super::merkle::MerklePath>,
+    pub proof: ::core::option::Option<super::smt::SmtOpening>,
```

Comment:
Are these opening's leaves supposed to always be of the `Single` variant (ie everything else would be an error)? If so, is there any benefit other than stronger typing to returning this as an opening instead of just the commitment?

Reply:
That's a great point. Yes, I think we can actually change the `AccountWitness` in base to `commitment: Digest` and `path: MerklePath` instead of `SmtProof`.

Thanks for the logs, I'll have a look.

Reply:
Changed AccountWitness to include the commitment and path directly, plus the account ID, which is necessary to be able to reconstruct an `SmtLeaf` from the other data.